contracts/end.cairo

```diff
@@ -896,12 +918,10 @@ func pack{
     }
 
     let (sender) = get_caller_address();
-    with_attr error_message("End/transfer-failed") {
-        let (claim) = _claim.read();
-        let (vow) = _vow.read();
-        let (success) = ClaimLike.transferFrom(claim, sender, vow, wad);
-        assert_not_zero(success);
-    }
+    // claim.burn(msg.sender, wad * RAY);
+    let (value) = mul(wad, Uint256(RAY, 0));
```

Comment:
How `RAY` is defined?

Reply:
10^27=0x33b2e3c9fd0803ce8000000, lower bits are not zero. Same problem in starknet-dss